### ISSUE TITLE
Not really an issue, but a question. How to disable target for friendly players?

### ISSUE THREAD

**Issue by tyrannicalduck** [Stanzilla/Kui_Stanzilla#2]
I couldn't find a place to just leave a comment, so I'm doing it here. How do I edit your kui.target-name.lua to only show hostile targets' targets. Running around the open world with players and having every single friendly player show his target is kinda crazy. 

I'm attaching a picture so kinda show what I mean.

Thanks!

![world of warcraft screenshot 2018 12 26 - 13 45 43 68](https://user-images.githubusercontent.com/22138145/50488087-d7350580-09ce-11e9-8205-1a821d7385ee.png)

**Comment by Stanzilla** [comment]
Yeah that's certainly something I want to do, just have to talk to @kesava on how to. My first idea was to ignore name-only mode but he said that's not easily doable.
